python/src/aac/util.py

```diff
@@ -155,3 +155,8 @@ def new_working_dir(directory):
     except Exception as e:
         chdir(current_dir)
         raise e
+
+
+def is_path_name(string: str) -> bool:
+    """Return True if string is a pathname; False, otherwise."""
+    return os.path.lexists(string)
```

Comment:
I would remove this; I don't think that the standard library name is too difficult to understand, and there isn't enough complexity abstracted away to warrant adding to `util.py`.  

I'm trying to keep the "utils" files slim, they're usually an indication that that codebase needs some organization and cleaning.